core/src/core/transaction.rs

```diff
@@ -290,9 +283,18 @@ impl Readable for TransactionBody {
 		let (input_len, output_len, kernel_len) =
 			ser_multiread!(reader, read_u64, read_u64, read_u64);
 
-		let inputs = read_and_verify_sorted(reader, input_len)?;
-		let outputs = read_and_verify_sorted(reader, output_len)?;
-		let kernels = read_and_verify_sorted(reader, kernel_len)?;
+		let inputs = read_and_verify_sorted(reader, input_len).map_err(|e| {
```

Comment:
I found a rule - don't log error if you don't handle it - very useful, especially if an error bubbles up and each layer logs it. It could be successfully handled, in this case we will just have lines of noise in the log or logged with the stack trace so we get all data in one place

Reply:
This was actually intentional (and temporary) as we try and track down the underlying issue for #1325.
Right now we're just getting a very generic (and context free) `CorruptedData` error propagating up.